docs/security/trust/content_trust.md

```diff
@@ -172,8 +172,8 @@ and push a tagged image. If this is the first time you have pushed an image
 using content trust on your system, the session looks like this:
 
 ```bash
-$ docker push docker/trusttest:latest
-The push refers to a repository [docker.io/docker/trusttest] (len: 1)
+$ docker push <username>/trusttest:testing
```

Comment:
Actually having read the rest of the doc, keeping it `testing` here and changing the note a little below would be great.